NAMESPACE

```diff
@@ -1,9 +1,93 @@
-export(sinkhorn_scale)
+# Generated by roxygen2: do not edit by hand
+
+export(DualSimplexSolver)
+export(add_basis_samples)
+export(add_default_anno)
+export(add_list_markers)
+export(add_noise)
+export(add_pure_samples)
+export(add_solution)
+export(add_solution_history)
+export(apply_default_filters)
+export(bool_filter)
+export(cat_markers)
+export(coerce_pred_true_basis)
+export(coerce_pred_true_props)
+export(concat_data)
+export(convert_sc_markers)
+export(create_simulation)
+export(describe_cat_features)
+export(filter_str)
+export(get_2d_subset)
+export(get_anno)
+export(get_fold_change)
+export(get_signature_markers)
+export(get_solution_history)
+export(guess_order)
+export(initialize_solution)
+export(is_logscale)
+export(linearize_dataset)
+export(log_dataset)
+export(n_sigma_filter)
+export(names_filter)
+export(nnls_C__)
+export(optim_config)
+export(optimize_solution)
+export(plot_basis_distribution)
+export(plot_errors)
+export(plot_feature)
+export(plot_feature_pair)
+export(plot_marker_enrichment)
+export(plot_negative_basis_change)
+export(plot_negative_proportions_change)
+export(plot_numeric_features)
+export(plot_projection_points)
+export(plot_proportions_distribution)
+export(plot_ptb_scatter)
+export(plot_ptp_lines)
+export(plot_ptp_scatter)
+export(plot_solution_history_anim)
+export(plot_svd_ds_matrix)
+export(predicate_filter)
+export(quantile_filter)
+export(read_gene_list)
+export(remove_zero_rows)
+export(replace_duplicate_genes)
 export(reverse_sinkhorn)
+export(reverse_solution_projection)
+export(reverse_solution_sinkhorn)
+export(reverse_svd_projection)
+export(revert_marker_list)
+export(set_anno)
+export(sinkhorn_scale)
 export(svd_project)
+export(threshold_filter)
+export(top_filter)
+export(transform_proj_umap)
+export(which_marker)
+import(Biobase)
+import(Matrix)
+import(RColorBrewer)
 import(Rcpp)
-import(R6)
-import(ggplot2)
+import(RcppArmadillo)
+import(combinat)
+import(cowplot)
 import(dplyr)
-import(Biobase)
+import(ggplot2)
+import(irlba)
+import(knitr)
+import(linseed)
+import(reshape2)
+import(scales)
+import(stringr)
+importFrom(Hmisc,describe)
+importFrom(Metrics,rmse)
+importFrom(R6,R6Class)
+importFrom(Rcpp,evalCpp)
+importFrom(grDevices,adjustcolor)
+importFrom(grid,textGrob)
+importFrom(gridExtra,arrangeGrob)
+importFrom(gridExtra,grid.arrange)
+importFrom(rlang,parse_expr)
+importFrom(tidyr,pivot_longer)
```

Comment:
Is this actually autogenerated? The lack of spaces after commas suggests the opposite, will it be preserved upon regeneration?

Reply:
It's actually just to call `function` instead of `package::function` inside the package. But in some cases, I'd say that we'd better use the `package::function` notation